packages/fela/src/bindings/connectFactory.js

```diff
@@ -59,7 +60,15 @@ export default function connectFactory(
         }
       }
 
-      return EnhancedComponent
+      // Re-attach static methods
+      EnhancedComponent.childContextTypes = component.childContextTypes;
```

Comment:
@rofrischmann It looks ok. Are we sure it's legit to hoist also all React statics?